src/features/projectsV2/ProjectsMap/ProjectMarkers/index.tsx

```diff
@@ -91,7 +96,7 @@ const ProjectMarkers = ({ categorizedProjects }: ProjectMarkersProps) => {
           visitProject={visitProject}
         />
       )),
-    [initiatePopupOpen, handleMarkerLeave, visitProject]
+    [initiatePopupOpen, handleMarkerLeave, visitProject, selectedClassification]
```

Comment:
1. Is `selectedClassification` needed as a dependency here?
2. Perhaps we should change `useMemo` to `useCallback` for `renderMarkers`? And memoize the results of the `renderMarkers` function for each type of marker?